src/Text/Pandoc/ImageSize.hs

```diff
@@ -116,7 +116,11 @@ imageType img = case B.take 4 img of
                      _                  -> mzero
 
 findSvgTag :: ByteString -> Bool
-findSvgTag img = B.null $ snd (B.breakSubstring img "<svg")
+findSvgTag img = prfx `elem` ["<svg ", "<svg\n", "<svg\r"]
+  where
+    prfx = B.take 5 $ last $ B.groupBy openingTag $ B.drop 7 img
+    -- B.groupBy openingTag matches first "<svg" or "<html" but not "<!--"
+    openingTag x y = x == '<' && y /= '!'
```

Comment:
@mb21, I added the tests. However, the first test does fail. I assume that the warning will be redirected to stderr?

Additionally, the code example does not compile:

```
pandoc/src/Text/Pandoc/ImageSize.hs:121:23: error:
    • Couldn't match expected type ‘ByteString’ with actual type ‘[t0]’
    • In the second argument of ‘($)’, namely
        ‘lastOrEmpty $ B.groupBy openingTag img’
      In the expression:
        B.take 5 $ lastOrEmpty $ B.groupBy openingTag img
      In an equation for ‘snip’:
          snip = B.take 5 $ lastOrEmpty $ B.groupBy openingTag img
```